src/Medidata.ZipkinTracer.Core/ZipkinClient.cs

```diff
@@ -35,10 +50,12 @@ public ZipkinClient(IZipkinConfig zipkinConfig, IOwinContext context, SpanCollec
 
             try
             {
-                spanCollector = collector ?? SpanCollector.GetInstance(
+                spanCollector = collector ?? GetInstance(
```

Comment:
@klettier There was a miss in unit test scenario for null collector. Can you please add one scenario to cover the GetInstance part? Thanks!